markerclusterer/src/markerclusterer.js

```diff
@@ -823,7 +855,7 @@ function Cluster(markerClusterer) {
   this.markers_ = [];
   this.bounds_ = null;
   this.clusterIcon_ = new ClusterIcon(this, markerClusterer.getStyles(),
```

Comment:
Why not have `getStyles` just include `commonStyleString_`, then no changes would be needed in `ClusterIcon`